blocks/v2-hero/v2-hero.css

```diff
@@ -481,6 +481,137 @@
 
 /* PDP - Hero section END */
 
+/* with engine stats variant START */
+
+.v2-hero--with-engine-stats {
+  height: unset;
+}
+
+.v2-hero--with-engine-stats picture {
+  background-color: rgb(29 29 29 / 5%);
+  display: block;
+  aspect-ratio: 1;
+}
+
+.v2-hero--with-engine-stats .v2-hero__image {
+  display: block;
+  position: unset;
+  object-fit: contain;
+  padding: 50px;
+}
+
+.redesign-v2 .v2-hero-container.section--with-background.reduced-height > picture {
+  position: absolute;
+  height: 100%;
+}
+
+@media (min-width: 990px) {
+  .v2-hero--with-engine-stats {
+    --stats-height: 160px;
+    --spacing-left: calc(calc(100vw - var(--wrapper-width)) / 2);
```

Comment:
no need of double calc. the things inside calc are already calculating things.
just remove the 2nd calc like this:
```
--spacing-left: calc((100vw - var(--wrapper-width)) / 2);
```